src/main/webapp/web.xml

```diff
@@ -98,7 +98,7 @@
     </servlet>
     <servlet-mapping>
         <servlet-name>delete-product</servlet-name>
-        <url-pattern>/products/all</url-pattern>
+        <url-pattern>/shopping-carts/products/delete</url-pattern>
```

Comment:
are you sure you changed mapping in the right place?

Reply:
oh, yep, I understood, fixed it